frontend/translations/ig.po

```diff
@@ -256,7 +256,7 @@ msgstr "Mepụta ọdịmara ọhụrụ"
 
 #: src/app/main/ui/settings/access-tokens.cljs
 #, unused
-msgid "dashboard.access-tokens.create-success"
+msgid "dashboard.access-tokens.create.success"
```

Comment:
Why this change?

Reply:
I don't really know when this change occurred but in some places it was with a period and in others with a hyphen. I eliminated the minority option. If you don't like it you can take the other one.